tests/test_client.py

```diff
@@ -0,0 +1,26 @@
+from pywps import Service
+from .common import client_for
+from finch.processes import get_processes
+import pytest
+
+
+class TestClient:
+    URL = "http://localhost:9099"
```

Comment:
Very minor suggestion, maybe allow override using environment var. So something like `URL = os.environ.get('WPS_URL', "http://localhost:9099")` which would allow to use the same test client to maybe target other Finch, ex the Finch in prod.

Reply:
Done.

Reply:
In fact this is a dummy value. It's not hitting the server running on this port, I'm creating a Service and getting the xml documents, then using this to instantiate client objects. I've changed the URL to `URL = "http://dummy_url.org"` to clarify this.